packages/commonwealth/client/scripts/views/pages/Communities/Communities.tsx

```diff
@@ -200,6 +209,8 @@ const CommunitiesPage = () => {
           </div>
           <IdeaLaunchpad />
         </div>
+        <TokensList />
```

Comment:
should we put it behind the featureflag?

Reply:
that check is inside this component

https://github.com/hicommonwealth/commonwealth/blob/d4582ba003e772ea297b0ce636af4d40001efc74/packages/commonwealth/client/scripts/views/pages/Communities/TokensList/TokensList.tsx#L36